src/components/BasicHead.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-<v-app-bar app style="height:64px;background-color:white;margin-left: 85px;">
+<v-app-bar app :style="appBarStyle">
```

Comment:
let's move these styles to the class `.vxg-app-bar`
then downstream apps can customize using css, rather than code

Reply:
Done